bears/go/GoErrCheckBear.py

```diff
@@ -0,0 +1,50 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.settings.Setting import path
+
+
+@linter(executable='errcheck',
+        output_format='regex',
+        output_regex=r'(?P<filename>[^:]+):(?P<line>\d+):'
+        r'(?P<col>\d+)\s*(?P<message>.*)',
```

Comment:
this is probably not a mistake, but from what i've seen on other bears, the ``r`` should be under the ``r`` on the previous line